tests/scattering1d/test_torch_scattering1d.py

```diff
@@ -434,7 +434,26 @@ def test_Q(device, backend):
     length = 1024
     shape = (length,)
 
+    # test different cases for Q
     with pytest.raises(ValueError) as ve:
         _ = Scattering1D(
-            J, shape, Q=0.9, backend=backend, frontend='torch').to(device)
+            J, shape, Q=0.9, backend=backend, frontend='torch')
     assert "Q should always be >= 1" in ve.value.args[0]
+
+    with pytest.raises(ValueError) as ve:
+        _ = Scattering1D(
+            J, shape, Q=[8], backend=backend, frontend='torch')
+    assert "Q must be an integer or a tuple" in ve.value.args[0]
+
+    Sc_int = Scattering1D(J, shape, Q=(8, ), backend=backend, frontend='torch')
+    Sc_tuple = Scattering1D(J, shape, Q=(8, 1), backend=backend, frontend='torch')
+
+    assert Sc_int.Q == Sc_tuple.Q
+
+    # test dummy input
+    x = torch.zeros(shape)
```

Comment:
Pls add `.to(device)` here

Reply:
You need to add a skip for the skcuda. Please look at the other tests to see how its done.

Reply:
A skip for the `skcuda` has been added. Thank, Muawiz! @MuawizChaudhary